src/EditorFeatures/Core/Implementation/Formatting/ICodeStyleConfigureService.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Formatting
+{
+    public interface ICodeStyleConfigureService : IWorkspaceService
```

Comment:
you probably not want this to be public type?

Reply:
fixed